imports/plugins/custom/gondor-theme/client/styles/navbar.less

```diff
@@ -0,0 +1,91 @@
+:root {
+  --component-bg: #52355A;
+  --component-color: #ffffff;
+  --cart-bg: #8B6096;
+}
+
+.rui.navbar {
+  background-color: var(--component-bg);
+  color: var(--component-color);
+}
+
+.rui.btn > div > i,
+.rui.btn > div > span {
+  color: var(--component-color) !important;
+}
+
+.navbar-item a:hover, 
+.navbar-item a:focus {
+  background-color: var(--cart-bg);
+}
+
+.caret {
+  color: var(--component-color);
+}
+
+.rui.navbar .accounts:hover,
+.rui.navbar .languages:hover, 
+.rui.navbar .currencies:hover,
+.rui.navbar .notification-icon:hover,
+.rui.navbar .search:hover,
+.rui.tagnav .navbar-item:hover > .rui.tag.link{
+  background-color: var(--cart-bg) !important;
+  color: var(--component-bg) !important;
+}
+
+.rui.navbar .accounts {
+  color: var(--component-bg) !important;
+}
+
+.rui.navbar a,
+.rui.navbar .dropdown-toggle {
+  color: var(--component-color);
+}
+
+a.brand,
+.navbar-item,
+.rui.tag.link:hover {
+  color: white !important;
+}
+
+.rui.navbar button{
+  color: var(--component-color) !important;
+}
+
+.rui.navbar .navbar-item:hover {
+  color: var(--component-color) !important;
+}
+
+// cart
+.cart-icon {
+  background-color: var(--cart-bg);
+}
+
+.cart-icon .badge {
+  background-color: var(--cart-bg);
+}
+
+.cart-icon:hover {
+  background-color: var(--component-bg);
+}
+
+// admin bar
+
+.rui.toolbar.admin-shortcut-bar {
+  background-color: var(--component-bg);
+}
+
+.rui.toolbar.admin-shortcut-bar .rui.btn-default.flat {
+  color: var(--component-color);
+}
+
+// menu dropdown bg
+.dropdown-menu > .active > a,
+.dropdown-menu > .active > a:hover,
+.dropdown-menu > .active > a:focus{
+  background-color: var(--component-bg);
+}
+
+.rui.tagnav .navbar-item:hover > .rui.tag.link {
+  color: white !important;
+}
```

Comment:
I think you should add a new line, because having a newline at the end of the file provides a simple check that the file has not been truncated.

Reply:
Thanks for the feedback. I will fix it